Resources/Maps/_Mono/Shuttles/Expedition/duran.yml

```
@@ -328,89 +390,109 @@ entities:
       data:
         0,0: 1085226772707934207
         0,-1: 18446743519709102080
-        -1,0: 16208736533883125759
+        -1,0: 16208733222463340543
         -1,-1: 18446709855168233472
         -2,-1: 9223372036854775808
-        -2,0: 141565355933888
+        -2,0: 141565343301824
         0,1: 15
         1,-1: 216454257090494464
-        1,0: 16974595
+        1,0: 16974081
+    - type: IsRoof
```

```suggestion
    - type: ImplicitRoof
```
so that entire grid has roof
the former is added to entities that roof the tile they're on